### PR TITLE
[WIP] Enable network integration tests via Zuul

### PR DESCRIPTION
**Pull request by gundalow**
Depends-On: https://review.openstack.org/602290

### CODE REVIEW

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

File: .zuul.d/jobs.yaml [hunk 1]
```diff
@@ -6,8 +6,7 @@
     #third-party-check:
     check:
       jobs:
-        - noop
-        #- ansible-test-network-integration-vyos-devel-py2
+        - ansible-test-network-integration-vyos-devel-py2
```
Comment (gundalow) [inline .zuul.d/jobs.yaml]: Update to remove `devel` as this is branched

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by TristanCacqueray** [comment]
You'll have to update the PR, it's currently stuck in zuul pipeline and it doesn't seems possible to dequeue a PR event, see: https://storyboard.openstack.org/#!/story/2003747

**Comment by TristanCacqueray** [comment]
I think the job can be fixed by https://review.openstack.org/602290 , i'll update the top comment to add a depends-on for test.

**Comment by TristanCacqueray** [comment]
recheck

**Comment by TristanCacqueray** [comment]
recheck

**Comment by TristanCacqueray** [comment]
recheck

**Comment by TristanCacqueray** [comment]
The Depends-on seems to be taken into account, the first node information is displayed, but the send one still fails with "The error was: 'dict object' has no attribute 'ansible_hostname'" ...

**Comment by TristanCacqueray** [comment]
recheck (PS4 of the depends'on)

**Comment by TristanCacqueray** [comment]
recheck

**Comment by TristanCacqueray** [comment]
Last PS4 adds "ignore_errors: yes" to the Print node information task. but the job stills fail, not sure the depends-on really work after all.

**Pull request closed without merging**